frontend/src/components/admin/users/UserManagementTableRow.tsx

```diff
@@ -0,0 +1,82 @@
+import {
+  Box,
+  Checkbox,
+  Icon,
+  Tr,
+  Td,
+  Text,
+  Tooltip,
+  useDisclosure,
+} from "@chakra-ui/react";
+import { MdMoreHoriz } from "react-icons/md";
+import React from "react";
+import ProfileDrawer from "./ProfileDrawer";
+
+type UserManagementTableRowProps = {
+  firstName: string;
+  lastName: string;
+  pronouns: string;
+  email: string;
+  phoneNumber: string;
+  checked: boolean;
+  onCheck: (params: any) => any;
```

Comment:
Can we make a TODO to change this type?
```suggestion
  // TODO: Replace param type
  onCheck: (params: any) => any;
```